nextjs/.gitignore

```diff
@@ -0,0 +1,2 @@
+.next
+node_modules
```

Comment:
Not that it matters too much, but I didn't see a .gitignore for the  expressjs example. Should there be?

Reply:
it didnt need one, didnt do anything special.